### PR TITLE
Show the bid history for a lot

### PR DESCRIPTION
**Pull request by orta**
This maps the event data into actual views :+1: 

Lets have a comment inline.

253c1f5137b

Danger: Ignore "Developer Specific file shouldn't be changed"

### CODE REVIEW

File: Artsy/Categories/Apple/UIViewController+SimpleChildren.m [hunk 1]
```diff
@@ -75,4 +75,11 @@ - (void)ar_removeChildViewController:(UIViewController *)controller
     [controller.view removeFromSuperview];
 }
 
+- (void)ar_modernRemoveChildViewController:(UIViewController *)controller
+{
+    [controller willMoveToParentViewController:nil];
+    [controller removeFromParentViewController];
+}
```
Comment (orta) [inline Artsy/Categories/Apple/UIViewController+SimpleChildren.m]: there isn't a function on ORStackView to deal with removing a ViewController, so we need a way to disconnect a viewController without dealing with the view hierarchy

Comment (sarahscott) [inline Artsy/Categories/Apple/UIViewController+SimpleChildren.m]: :+1:

**Comment by sarahscott** [comment]
Looks good to me - restarted the circle build because the fail didn't seem legit and/or danger-related

**Comment by orta** [comment]
fail is legit, will ship a new version soon

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Errors</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>Developer Specific file shouldn&#39;t be changed</td>
    </tr>
        <tr>
      <td>:no_entry_sign:</td>
      <td>There were <a href="https://eigen-ci.s3.amazonaws.com/1799/index.html">snapshot errors</a></td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Warning</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARArtworkFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkFavoritesNetworkModelSpec</a> in <code>test_getFavorites__makes_request_if_no_request_is_in_progress</code></li>
<li><a href="https://github.com/search?q=ARArtworkViewControllerBuyButton+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkViewControllerBuyButtonSpec</a> in <code>test_buy_button__posts_order_if_artwork_has_no_edition_sets</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__passes_artwork_to_share_controller</code></li>
<li><a href="https://github.com/search?q=ARFairViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARTopMenuViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARTopMenuViewControllerSpec</a> in <code>test_navigation__favorites__logged_in__tab_behavior__already_selected__pops_to_root</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
OK, should be green now

**Comment by sarahscott** [comment]
:green_apple:

**Pull request merged**